tests/test_unclassified_job.py

```diff
@@ -21,6 +21,8 @@ def test_unclassified_failure(self, mozwebqa):
         # Open resultset page and search for next unclassified failure
         resultset_page = ResultsetPage(mozwebqa)
         resultset_page.go_to_page()
+        Assert.greater_equal("resultset_page.return_unclassified_failure_count", "1")
```

Comment:
This isn't returning the unclassified failure count from the page, it's checking that the string "resultset_page.return_unclassified_failure_count" is greater than the string "1". This happens to be true because 'f' has a higher value than '1'. See https://docs.python.org/2/tutorial/datastructures.html#comparing-sequences-and-other-types for details on how string comparisons are made.

To fix this, we need to call `resultset_page.return_unclassified_failure_count()` and compare it to `1` (not a string).